unity-renderer/Assets/DCLServices/WearablesCatalogService/WearablesWebInterfaceBridge.cs

```diff
@@ -0,0 +1,13 @@
+﻿using DCL.Interface;
+
+namespace DCLServices.WearablesCatalogService
+{
+    public class WearablesWebInterfaceBridge
+    {
+        public virtual void RequestWearables(string ownedByUser, string[] wearableIds, string[] collectionIds, string context) =>
```

Comment:
Why these methods are virtual?

Reply:
I left them as `virtual` in order to be able to mock them in the tests (that are not still implemented)

Reply:
I think it's a proper moment to introduce some tests. Otherwise, you will carry potential bugs to the next PR that is connected to the usage of the API only.

Reply:
Yep, I agree. My idea was to start implementing tests just after applying all the feedback here and before starting the second part of the refactor 👍